build.sbt

```diff
@@ -11,6 +11,7 @@ val http4sVersion = "0.21.4"
 lazy val specs2Http4s = (project in file("."))
   .settings(
     name := "specs2-http4s",
+    description := "specs2 matchers for http4s",
     organization := "org.specs2",
     version := specs2Version,
```

Comment:
The last question I have before releasing is how we ought to version this. Do you want it to stay in lock-step with the main repo, or have it be independent? If the latter, shall we start from 0.1.0 to allow for some API flexibility before an eventual major version?

Reply:
I think it's better to leave it independent. About the version number, since it was already out there it might as well start as 1.0.0. But I don't have strong preferences, do as you feel.

Reply:
Yeah, I'll go with 1.0.0 since that makes sense :+1: